src/test/java/seedu/address/storage/JsonAdaptedPetTest.java

```diff
@@ -0,0 +1,116 @@
+package seedu.address.storage;
+
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import seedu.address.model.InvalidPetException;
+
+public class JsonAdaptedPetTest {
+    public static final String INVALID_PET_NAME = "@.@";
+    public static final String INVALID_LASTDONETASKTIME = "123456";
+    public static final String INVALID_LEVEL_UPPER = "4";
+    public static final String INVALID_LEVEL_LOWER = "0";
+    public static final String INVALID_EXP_EXP_FIRST = "150";
+    public static final String INVALID_EXP_LEVEL_FIRST = "1";
+    public static final String INVALID_EXP_EXP_SECOND = "50";
+    public static final String INVALID_EXP_LEVEL_SECOND = "2";
+    public static final String INVALID_MOOD = "NOT UNHAPPY";
+
+    public static final String VALID_PET_NAME = "Momu";
+    public static final String VALID_LASTDONETASKTIME = "2020-04-11T13:59:24.013541600";
+    public static final String VALID_LEVEL = "2";
+    public static final String VALID_EXP_EXP = "150";
+    public static final String VALID_EXP_LEVEL = "2";
+    public static final String VALID_MOOD = "HANGRY";
+
+    @Test
+    public void toModelType_invalidPetName_returnsInvalidPetException() {
```

Comment:
I like how you added a JsonAdaptedPetTest for reading the pet correctly, good job!